index.html

```diff
@@ -35,41 +35,315 @@
+                <li class="nav__item">
+                  <a class="nav__link" href="#">Services</a>
                 </li>
-                <li class="first-screen__nav-item">
-                  <a class="first-screen__nav-link" href="#">Specials</a>
+                <li class="nav__item">
+                  <a class="nav__link" href="#">Specials</a>
                 </li>
-                <li class="first-screen__nav-item">
-                  <a class="first-screen__nav-link" href="#">Facilities</a>
+                <li class="nav__item">
+                  <a class="nav__link" href="#">Facilities</a>
                 </li>
-                <li class="first-screen__nav-item">
-                  <a class="first-screen__nav-link" href="#">Reviews</a>
+                <li class="nav__item">
+                  <a class="nav__link" href="#">Reviews</a>
                 </li>
-                <li class="first-screen__nav-item">
-                  <a class="first-screen__nav-link" href="#">Shop</a>
+                <li class="nav__item">
+                  <a class="nav__link" href="#">Shop</a>
                 </li>
-                <li class="first-screen__nav-item">
-                  <a class="first-screen__nav-link" href="#">About</a>
+                <li class="nav__item">
+                  <a class="nav__link" href="#">About</a>
                 </li>
               </ul>
             </nav>
         </header>
         <div class="first-screen__title-wrap">
-          <h1 class="first-screen__title">Premier dog boarding, daycare & day spa</h1>
+          <h1 class="title">Premier dog boarding, daycare & day spa</h1>
           <button type="button" name="button" class="first-screen__button">
-            <a href="" class="first-screen__button-link">Make an apointment</a>
+            <a href="" class="button__link">Make an apointment</a>
+          </button>
+        </div>
+      </div>
+    </section>
+    <section class="info-about-us">
+      <div class="info-about-us__container container">
+        <div class="info-about-us__who-we-are">
+          <div class="icon-with-subtitle">
+            <img class="icon-with-subtitle__icon" src="./img/rhombus_icon.svg" alt="rhombus-icon">
+            <p class="icon-with-subtitle__subtitle">Who we are</p>
+          </div>
+          <div class="info-about-us__title-wrap">
+            <h2 class="title">Nemo enim ipsam voluptatem quia voluptas in proident</h2>
+          </div>
+          <div class="info-about-us__text-wrap">
+            <p class="text text--gray">Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam,
+            nisi ut aliquid ex ea commodi consequatur. Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam
+            nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur.</p>
+          </div>
+
+
+          <img class="dog-eyes" src="./img/dog_eyes.svg" alt="Dog eyes">
+        </div>
```

Comment:
```suggestion
        <div class="info-about-us__wrap">
          <p class="info-about-us__subtitle">Who we are</p>
          <h2 class="title">Nemo enim ipsam voluptatem quia voluptas in proident</h2>
          <p class="info-about-us__description">Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam,
            nisi ut aliquid ex ea commodi consequatur. Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam
            nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur.</p>
          <img class="info-about-us__img" src="./img/dog_eyes.svg" alt="Dog eyes">
        </div>
```
іконку добавити через :before